styles/stylesheet.css

```diff
@@ -0,0 +1,376 @@
+/* Body and general styling */
+body {
+  background-color: #ffffff;
+  color: #000000;
+  padding: 0;
+  border: 0;
+  margin: 0;
+}
+
+h3 {
+  color: #3b6860;
+  font: 2rem "Irish Grover", cursive;
+}
+.hidden {
+  display: none;
+}
+h4 {
+  color: #3b6860;
+  font: 1.6rem "Irish Grover", cursive;
+}
+h5 {
+  color: #3b6860;
+  font: 1.3rem "Irish Grover", cursive;
+  margin:1rem;
+}
+p {
+  font: 1rem/1.5rem Arial, Helvetica, sans-serif;
```

Comment:
clever use of this. I honestly had to lookup why you did the 1rem/1.5rem. It's good shorthand, but not the most readable thing.